test/provider_devdax_memory.cpp

```diff
@@ -0,0 +1,249 @@
+// Copyright (C) 2023 Intel Corporation
```

Comment:
2024

Reply:
Done

Reply:
@lukaszstolarczuk Please verify and resolve.

Reply:
@lukaszstolarczuk ping